vectorstores/cloudsql/loader/documentloader.go

```diff
@@ -0,0 +1,441 @@
+package loader
```

Comment:
Move this package to https://github.com/averikitsch/langchaingo/tree/main/documentloaders